src/main/java/com/shrralis/ssdemo1/exception/interfaces/AbstractShrralisException.java

```diff
@@ -0,0 +1,33 @@
+package com.shrralis.ssdemo1.exception.interfaces;
+
+import com.shrralis.tools.model.JsonError;
+
+/**
+ * Interface of exceptions that can give us some {@link com.shrralis.tools.model.JsonError.Error}
+ *
+ * @author shrralis (https://t.me/Shrralis)
+ * @version 1.0 Created 12/24/17 at 2:19 AM
+ */
+public abstract class AbstractShrralisException extends Exception {
```

Comment:
What does Shrralis mean?

Reply:
I had no idea how to call that. Now we decided to rename it as `Abstract Citizen Exception`. Waiting for that with coming commits.